code/game/objects/items/weapons/tools.dm

```diff
@@ -305,66 +295,75 @@
 	if (istype(location, /turf))
 		location.hotspot_expose(700, 5)
 
-/obj/item/weldingtool/attack(mob/living/M as mob, mob/user as mob, var/target_zone)
-
-	if(hasorgans(M))
+/obj/item/weldingtool/attack(mob/living/M, mob/user, var/target_zone)
```

Comment:
Don't we need to cover the case where the target limb has no damage? 

Otherwise you get:
- User clicks with welder on limb that isn't damaged.
- "You finish repairing the thing."
- But you didn't actually repair anything.